boards/arm/stm32/stm32f411e-disco/Kconfig

```diff
@@ -7,7 +7,7 @@ if ARCH_BOARD_STM32F411E_DISCO
 
 config STM32F411DISCO_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1024
```

Comment:
**It is inappropriate to increase the size of this tuned stack size.  Let's not do that.**